homeassistant/components/zwave_js/services.py

```diff
@@ -94,25 +95,24 @@ def async_register(self) -> None:
         def get_nodes_from_service_data(val: dict[str, Any]) -> dict[str, Any]:
             """Get nodes set from service data."""
             nodes: set[ZwaveNode] = set()
-            try:
-                if ATTR_ENTITY_ID in val:
-                    nodes |= {
+            for entity_id in val.pop(ATTR_ENTITY_ID, []):
+                try:
+                    nodes.add(
                         async_get_node_from_entity_id(
                             self._hass, entity_id, self._ent_reg, self._dev_reg
                         )
-                        for entity_id in val[ATTR_ENTITY_ID]
-                    }
-                    val.pop(ATTR_ENTITY_ID)
-                if ATTR_DEVICE_ID in val:
-                    nodes |= {
+                    )
+                except ValueError as err:
+                    const.LOGGER.warning(err.args[0])
```

Comment:
Are we OK with this message being a warning? Or should we make this an `info`?

Reply:
I think warning is appropriate since we've detected invalid service data.